doc/source/whatsnew/v1.3.0.rst

```diff
@@ -140,6 +140,9 @@ properly format HTML and eliminate some inconsistencies (:issue:`39942` :issue:`
 
 :class:`.Styler` has also been compatible with non-unique index or columns, at least for as many features as are fully compatible, others made only partially compatible (:issue:`41269`).
 
+We have added an extension to allow LaTeX styling as an alternative to CSS styling and a method :meth:`.Styler.to_latex`
```

Comment:
can you link to examples in io.rst

Reply:
added an issue. will be follow on. #41613